br-ext/package/optee_test_ext/Config.in

```diff
@@ -24,4 +24,12 @@ config BR2_PACKAGE_OPTEE_TEST_EXT_CROSS_COMPILE
 	help
 	  some help
 
+config BR2_PACKAGE_OPTEE_TEST_EXT_GP_PACKAGE
+	string "GP package"
+	default ""
+	help
+	  Path to the GlobalPlatform package containing description files
+	  and TAs needed to build the complete GP test suite. The package
+	  is not free and has to be acquired directly from GlobalPlatform.
```

Comment:
Minor: you could be more precise here. The exact name of the package is "TEE Initial Configuration Test Suite with Excluded Tests List v2.0.0.2" and the URL is https://globalplatform.org/test-suites/tee-initial-configuration-test-suite-with-excluded-tests-list/. Also, the package is free for GP members and can be purchased by non-members. 

For clarity: I understand it is the path to the compressed archive (.7z file), no need to extract it first, right?

Reply:
> For clarity: I understand it is the path to the compressed archive (.7z file), no need to extract it first, right?

Correct.